boards/arm/mimxrt595_evk/mimxrt595_evk_cm33.dts

```diff
@@ -367,6 +367,10 @@ arduino_serial: &flexcomm12 {
 	status = "okay";
 };
 
+&dma1 {
```

Comment:
Rather than enabling the DMA1 node at the board level, could we enable it for a specific testcase? I understand the value of enabling the DMA node to make sure the driver can build against it, but I don't want to enable another instance of the DMA driver in every build for the RT595 where CONFIG_DMA=y.

Reply:
true, alright I have updated this.